Tutorials/Add-Google-Authentication/index.md

```diff
@@ -20,17 +20,17 @@ Developers who won't mind configuring a Google API, running the solution in Visu
 
 ### Prerequisites
 Here is what you will need for this tutorial:
-- Visual studio installed
-- A Google account
-- A working Umbraco solution
+- [Visual Studio](https://visualstudio.microsoft.com/) installed
+- A [Google](https://www.google.com/intl/en-GB/gmail/about/#) account
```

Comment:
You point to a gmail account.  The correct link for a google account is https://myaccount.google.com/

Reply:
Hey @dampee thanks for suggesting the changes. Have corrected them accordingly. Please let me know in case any other additions required

Umbraco - https://our.umbraco.com/member/275491